include/deal.II/fe/fe.h

```diff
@@ -1766,7 +1766,7 @@ class FiniteElement : public Subscriptor, public FiniteElementData<dim>
    */
   std::pair<std::pair<unsigned int, unsigned int>, unsigned int>
   face_system_to_base_index(const unsigned int index,
-                            const unsigned int face_no = 0) const;
+                            const unsigned int face_no) const;
```

Comment:
Do we need to document this change somewhere as this can have user-visible effects?

Reply:
This was not intended. I have removed the default value to be able to identify all instances where these functions are used. Reverted!

Reply:
> This was not intended.

That's why we have code review :wink: